ChangeLog

```diff
@@ -1,3 +1,428 @@
+
+- Features added since Pacemaker-2.1.6
+  + CIB: deprecate "ordering" attribute of "resource_set"
+  + CIB: new cluster option "node-pending-timeout" (defaulting to 2 hours) to
+    fence nodes that do not join Pacemaker's controller group within this much
+    time after joining the cluster
+  + controller: PCMK_node_start_state now works with Pacemaker Remote nodes
+  + crm_verify: add --quiet option (currently same as default behavior)
+  + tools: crm_node supports standard --output-as/--output-to arguments
+  + tests: CTSlab.py was renamed to cts-lab
+
+- Fixes since Pacemaker-2.1.6
+  + logging: restore ability to enable XML trace logs by file and function
+    (regression introduced in 2.1.6)
+  + scheduler: avoid double free with disabled recurring actions
+    (regression introduced in 2.1.5)
+  + tools: consider dampening argument when setting values with attrd_updater
+    (regression introduced in 2.1.5)
+  + tools: wait for reply from crm_node -R (regression introduced in 2.0.5)
+  + attrd: avoid race condition at writer election
+  + CIB: be more strict about ignoring colocation elements without an ID
+  + controller: do not check whether watchdog fencing is enabled
+    if "stonith-watchdog-timeout" is not configured
+  + controller: don't try to execute agent action at shutdown
+  + controller: avoid race condition when updating node state during join
+  + controller: correctly determine state of a fenced node without a name
+  + controller: wait a second between fencer connection attempts
+  + libpacemaker: avoid shuffling clone instances unnecessarily
+  + libpacemaker: get bundle container's promotion score from correct node
+  + libpacemaker: role-based colocations now works with bundles
+  + libpacemaker: clone-node-max now works with cloned groups
+  + scheduler: compare anti-colocation dependent negative preferences against
+    stickiness
+  + scheduler: consider explicit colocations with group members
+  + scheduler: avoid fencing a pending node without a name
+  + scheduler: properly evaluate rules in action meta-attributes
+  + scheduler: properly sort rule-based blocks when overwriting values
+  + tools: avoid showing pending nodes as having "<3.15.1" feature set in
+    crm_mon
+  + tools: fix display of clone descriptions
+  + tools: crm_resource now reports an error rather than time out when trying
+    to restart an unmanaged resource
+  + tools: crm_resource now properly detects which promoted role name to use
+    in ban and move constraints
+
+- Public API changes since Pacemaker-2.1.6
+  + libcib: cib_t now supports transactions via new cib_api_operations_t
+    methods, new cib_transaction value in enum cib_call_options, and new
+    cib_t transaction and user members
+  + libcib: cib_t now supports setting the ACL user for methods via new
+    cib_api_operations_t set_user() method
+  + libcib: deprecate cib_api_operations_t methods inputfd(), noop(), quit(),
+    set_op_callback(), and signon_raw(), and
```

Comment:
And what?

Reply:
ah, leftover from shuffling things a bit